erpnext/public/js/financial_statements.js

```diff
@@ -33,7 +33,7 @@ erpnext.financial_statements = {
 
 			return value;
 		} else if (frappe.query_report.get_filter_value("selected_view") == "Margin" && data) {
-			if (column.fieldname == "account" && data.account_name == __("Income")) {
+			if (column.fieldname == "stub" && data.account_name == __("Income")) {
```

Comment:
Chaging fieldname to 'stub' breaks export as the report still generates with 'account' as fieldname.

Reply:
Is there a way to fix this via a pre-export hook, where necessary?

In the javascript code `account` is misleading, at best: it's the table stub column which may contain anything, including, but not exclusively, an account.

Reply:
> In the javascript code account is misleading

Not sure how this is misleading. Any Examples?

Reply:
`account` is misleading in that its contents are not limited to account strings. In reality, though, it's the table's stub column, which indicates what the row to the right contains, including:
- accounts
- summary lines
- component descriptions / title / section separators